common/Tests/Framework/src/AITestClass.ts

```diff
@@ -24,6 +24,37 @@ function _getObjName(target:any, unknownValue?:string) {
     return (((target || {})["constructor"]) || {}).name || unknownValue || "";
 }
 
+function _getAllAiDataKeys<T = any>(target: T, callbackfn: (name: string, value: T[keyof T]) => void) {
```

Comment:
Base test class updates which will be used validate and remove any registered events, hooks etc.